lib/ae_mdw/sync/async_tasks/derive_aex9_presence.ex

```diff
@@ -0,0 +1,124 @@
+defmodule AeMdw.Sync.AsyncTasks.DeriveAex9Presence do
+  @moduledoc """
+  Async work to derive AEX9 presence from balance dry-running
+  from a create contract.
+  """
+  @behaviour AeMdw.Sync.AsyncTasks.Work
+
+  alias AeMdw.Node.Db, as: DBN
+
+  alias AeMdw.Contract
+  alias AeMdw.Db.Contract, as: DBContract
+  alias AeMdw.Db.Model
+  alias AeMdw.Db.Util, as: DBU
```

Comment:
We could perhaps get rid of renaming aliases, so that we can find all function for any function more easily

Reply:
It's used through out the code probably to allow accessing `AeMdw.Contract` and `AeMdw.Db.Contract` (same for Util in original module but here I could change for Util).

I got used to the DBN but we can have a refactoring PR to get rid of it through out the code renaming to sth like `AeMdw.Node.NodeDb`.